src/inc-js-variants.php

```diff
@@ -42,120 +42,52 @@
 define('AJAX_NO_AUTH', '8');
 define('AJAX_DATA_ERROR', '9');
 
-$_SETT = array('objectid_length' => array('transcripts' => 8));
+$_SETT = array(
+    'objectid_length' => array('transcripts' => 8),
+    'variant_validator' => array(
+        'genome_builds' => array('hg19', 'hg38')
```

Comment:
Two requests;
- Better stick to the existing `$_SETT` structure, so the code in this file doesn't look different from the code in other files trying to reach the same information.
- Better add a comment to `inc-init.php` in the relevant place that we need to check this file if we ever change the builds. The same is already done for the `objectid_length` setting, so you can copy that format.